page_objects/login_page.js

```diff
@@ -1,31 +1,84 @@
-// const { element } = require("protractor");
 let Input = require("../elements/input.element")
 let Button = require("../elements/button.element")
 let TextView = require("../elements/text.view")
+let Dropdown = require("../elements/dropdown.element")
 
-let BasePage = require ("../base/base.page")
+let BasePage = require("../base/base.page")
 
 let emailLocator = by.css('#email');
 let passwordLocator = by.css('#passwd');
 let submitbtnLocator = by.css('#SubmitLogin');
 let textloginLocator = by.css('#login_form>.page-subheading');
 let errorpassword = by.css('.alert li')
+let emailforRegLocator = by.css('#email_create')
+let createAnAccountLocator = by.css('#SubmitCreate')
+let submAccountLocator = by.css('#submitAccount')
+//let textPersonalInfLocator = by.xPath("//*[.='Your personal information']")
+let firstNameTitleLocator = by.css('#customer_firstname')
+let lastNameTitleLocator = by.css('#customer_lastname')
+let firstNameAddLocator = by.css('[name=firstname]')
+let lastNameAddLocator = by.css('[name=lastname]')
+let addressLocator = by.css('[name=address1]')
+let cityLocator = by.css('[name=city]')
+let stateLocator = by.css('#id_state')
+let postcodeLocator = by.css('#postcode')
+let countryLocator = by.css('#id_country')
+let mobileLocator = by.css('#phone_mobile')
+let addressAliasLocator = by.css('#alias')
 
-class LoginPage extends BasePage {   
-    async login (email,password) {
+let regForm = by.xpath('//label[.="Title"]')
+let successfulReg = by.css('.info-account')
+
+class LoginPage extends BasePage {
+    async login(email, password) {
         await allure.createStep(`Logging in ${email} / ${password}`, async () => {
             await this.getEmailInput().sendKeys(email);
             await this.getPassInput().sendKeys(password);
             await this.getSubmButInput().click();
         })()
     }
-    async getErrorMessage () {
-        return await this.getErrorElement().getText(); 
-    }   
-    async clearCredentials () {
-         await this.getEmailInput().clear();
-         await this.getPassInput().clear();
+    async getErrorMessage() {
+        return await this.getErrorElement().getText();
+    }
+    async sendEmail(email) {
+        await allure.createStep(`Sign in ${email}`, async () => {
+            await this.getEmailInputForReg().sendKeys(email);
+        })()
+    }
+    async createAnAccout() {
+        await this.getSubmButForRegInput().click();
+    }
+    async getTextFromRegisterForm() {
+        await this.getBaseElementForReg().click();
+    }
+    async getTextForRegForm() {
+        return await this.getTitleforExpect().getText();
+    }
+    async getTextForSuccessReg() {
```

Comment:
After the user registered, it is no more login screen showed. This method should be moved to the new page object class. 

If you see the page on UI is changed (the same should be in your source code) page objected class should be also changed

Reply:
fixed